src/jit/lower.cpp

```diff
@@ -1357,10 +1367,80 @@ void Lowering::LowerArg(GenTreeCall* call, GenTreePtr* ppArg)
         // If an extra node is returned, splice it in the right place in the tree.
         if (arg != putArg)
         {
-            ReplaceArgWithPutArgOrCopy(ppArg, putArg);
+            ReplaceArgWithPutArgOrBitcast(ppArg, putArg);
+        }
+    }
+}
+
+#ifdef _TARGET_ARMARCH_
+//------------------------------------------------------------------------
+// LowerFloatArg: Lower the float call argument on the arm platform.
+//
+// Arguments:
+//    arg    - The arg node
+//    info   - call argument info
+//    argNum - argument number
+//
+//
+// Return Value:
+//    Return nullptr, if no transformation was done;
+//    return arg if there was in place transformation;
+//    return a new tree if the root was changed.
+//
+GenTreePtr Lowering::LowerFloatArg(GenTreePtr arg, fgArgTabEntryPtr info, unsigned argNum)
```

Comment:
I believe it is the case that we are moving away from `GenTreePtr` to `GenTree*`.

Reply:
I have not heard about it, do we?

Reply:
This has come up a few times. Here's a relatively recent reference: https://github.com/dotnet/coreclr/pull/14020#discussion_r139248905

We should probably put it in the coding conventions. @BruceForstall what do you think?

Reply:
I agree that it's weird that GenTreePtr is an oddball. Maybe if we want to stop using it, we should agree and then do a copy/replace everywhere. Generally, though, I'm ambivalent about it.

Reply:
Will change it before the merge, do not want to rerun tests now.